twilight-model/src/voice/mod.rs

```diff
@@ -1,11 +1,15 @@
+pub mod payload;
 pub(crate) mod voice_state;
 
 mod close_code;
+mod encryption_mode;
 mod opcode;
+mod speaking_flags;
 mod voice_region;
 
```

Comment:
Can we add a version here, something like:
```rust
/// Voice gateway version Twilight supports.
pub const VOICE_GATEWAY_VERSION: u8 = 4;
```

Reply:
Hmm, maybe these models should just in their entirety belong inside of the voice crate (since no other crate would use them)?

Reply:
Yeah maybe it is a better idea